contributing.rst

```diff
@@ -1,7 +1,49 @@
-The git stuff
+The Git stuff
 =============
 
-`Guide by github, scroll down a bit <https://guides.github.com/activities/contributing-to-open-source/>`_
+General commit rules
+--------------------
+
+1. All changes introduced by the commit (diff lines) must be related to the
```

Comment:
@reactormonk I am sorry, but I don't understand your suggestion, I should change what to "commits"? I am not talking about PR, but rules for single commit, the subject is first line of the commit description.